### PR TITLE
Docs small fixes

### PR DESCRIPTION
**Pull request by rrecuero**
@spalladino

### CODE REVIEW

File: packages/docs/docs/docs/deploying.md [hunk 1]
```diff
@@ -56,14 +56,20 @@ by now, inside the `my-project` directory you should have a `package.json` file
 a `truffle-config.js` file (created by the `zos` CLI for Truffle), and a
 `zos.json` file (created by `zos` for ZeppelinOS).
 
+Note: if you haven't installed Truffle, run the following command:
+
+```console
+npm install -g truffle
```
Comment (spalladino) [inline packages/docs/docs/docs/deploying.md]:
```suggestion
npm install truffle
```

I wouldn't suggest globall install until https://github.com/zeppelinos/zos/issues/579 is fixed

File: packages/docs/docs/docs/deploying.md [hunk 2]
```diff
@@ -56,14 +56,20 @@ by now, inside the `my-project` directory you should have a `package.json` file
...
+
 ## Adding a contract
 
 Let's create a very simple contract as an example to be added to the project.
 Name it `MyContract.sol`, and put it in the `contracts` folder with the
 following Solidity code:
 
 ```solidity
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.0;
```
Comment (spalladino) [inline packages/docs/docs/docs/deploying.md]: Agree on migrating to 0.5, but I'd make sure that we update all code examples in our docs, to avoid having some contracts on 0.4 and others on 0.5. I'm not sure if there are any others around, though.

File: packages/docs/docs/docs/deploying.md [hunk 3]
```diff
@@ -127,27 +133,27 @@ To start working with it, open a separate terminal and run:
...
+In this case, we are telling it to work with the `local` network with the
+`--network` option, and also setting a default sender address for the
+transactions we will run with the `--from` option. Additionally, the
+`expires` flag allows us to indicate the session expiration time in seconds.
+
+> Note that we are using a specific address for the `--from` option which
+is different to the default address that `ganache-cli` would use.
+This is because we need to use different addresses in order to create
+upgradeable contracts and to query them. This problem is
+known as the "transparent proxy issue" and you can read more about it in the
```
Comment (spalladino) [inline packages/docs/docs/docs/deploying.md]: Should we rephrase it to avoid using "issue", since we're at it?

Comment (rrecuero) [reply packages/docs/docs/docs/deploying.md]: sure thing

File: packages/docs/docs/docs/deploying.md (hunk 2 above)
Comment (rrecuero) [reply packages/docs/docs/docs/deploying.md]: fixed the others
